src/container/RelayContainer.js

```diff
@@ -300,7 +301,7 @@ function createContainerComponent(
               }
             });
             if (callback) {
-              callback.call(this.refs.component, {...readyState, mounted});
+              callback.call(this.refs.component || null, {...readyState, mounted});
```

Comment:
Pass null to callback if we have no reference to a stateless functional component.